src/functions-and-arrays.js

```diff
@@ -52,18 +144,27 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
-
+function uniquifyArray(arrayOfRandomWords) {
+  return arrayOfRandomWords.filter((item, index) => arrayOfRandomWords.indexOf(item) === index);
+}
 
+console.log(uniquifyArray(wordsUnique));
 
-// Iteration #6: Find elements
+// Iteration #6: Find elements √
 const wordsFind = ['machine', 'subset', 'trouble', 'starting', 'matter', 'eating', 'truth', 'disobedience'];
 
-function doesWordExist() {}
-
+function doesWordExist(inputString, inputArray) {
+  for (let i = 0; i < inputArray.length; i++) {
```

Comment:
Nice job here! Now, can you solve this one with an array method? Bet you can even turn it into a one liner solution! 😄 🎉